### DISCUSSION TITLE
SVGs inside Selects

### DISCUSSION THREAD

**Discussion by eyyyyyyy3** [saadeghi/daisyui#2569] [Q&A] [answered]
Does daisyUI allow for adding svg files as option fields inside a select? I know that it doesn't work with plain HTML selects, at least not without some hacking.

**Comment by saadeghi** [comment, marked as answer]
SVG can't be inside the `<input>` element of course, but you can do one of the followings:

1. You can use daisyUI `input` class on a div and put an `<input>` inside it alongside with the icon:  
  https://play.tailwindcss.com/d257vfRbR2
2. You can use a relative div, put the input and icon in it and change the icon position to `absolute`
3. You can set a background picture for the input pointing to an SVG image file

Let me know if you have a question.
